src/modules/settingsPage.js

```diff
@@ -687,18 +682,18 @@ query{
 					data = JSON.parse(evt.target.result)
 				}
 				catch(e){
-					alert("error parsing JSON")
+					alert(translate("$settings_import_parser_error"))
```

Comment:
Should probably be a more genering key, used in all places where JSON parsing can go wrong, like "$error_JSONparsing"